docs/admin/authorization/rbac.md

```diff
@@ -58,17 +63,20 @@ metadata:
   # "namespace" omitted since ClusterRoles are not namespaced.
   name: secret-reader
 rules:
-  - apiGroups: [""]
-    resources: ["secrets"]
-    verbs: ["get", "watch", "list"]
+- apiGroups: [""]
+  resources: ["secrets"]
+  verbs: ["get", "watch", "list"]
 ```
 
-`RoleBindings` perform the task of granting the permission to a user or set of
-users. They hold a list of subjects which they apply to, and a reference to the
-`Role` being assigned.
+### RoleBindings and ClusterRoleBindings
+
+A role binding grants the permissions defined in a role to a user or set of users.
+It holds a list of subjects (users, groups, or service accounts), and a reference to the role being granted.
+Permissions can be granted within a namespace with a `RoleBinding` object, or cluster-wide with a `ClusterRoleBinding` object.
 
-The following `RoleBinding` assigns the "pod-reader" role to the user "jane"
-within the "default" namespace, and allows jane to read pods.
+A `RoleBinding` may reference a `Role` in the same namespace.
+The following `RoleBinding` grants the "pod-reader" role to the user "jane" within the namespace "default".
```

Comment:
I think punctuation inside quotes looks odd when the quotes are indicating a siginficant token, but just reworked the order to avoid the issue :)